src/coreclr/build-runtime.cmd

```diff
@@ -734,7 +546,7 @@ echo -cmakeargs: user-settable additional arguments passed to CMake.
 echo -configureonly: skip all builds; only run CMake ^(default: CMake and builds are run^)
 echo -skipconfigure: skip CMake ^(default: CMake is run^)
 echo -skipnative: skip building native components ^(default: native components are built^).
-echo -skipcrossarchnative: skip building cross-architecture native components ^(default: components are built^).
+echo -SkipCrossTargetBuildNative: skip building cross-architecture native components ^(default: components are built^).
```

Comment:
This isn't processed by this script; it is processed by something this passes it to? (cmake?)